brushfire-core/src/main/scala/com/stripe/brushfire/Brushfire.scala

```diff
@@ -42,14 +42,23 @@ trait Splitter[V, T] {
 }
 
 /** Candidate split for a tree node */
-trait Split[V, T] {
-  def predicates: Iterable[(Predicate[V], T)]
+case class Split[V, T](predicate: Predicate[V], leftDistribution: T, rightDistribution: T) {
+
+  /**
+   * Given a feature key, create a SplitNode from this Split.
+   *
+   * Note that the leaves of this node will likely need to be
+   * renumbered if this node is put into a larger tree.
+   */
+  def createSplitNode[K](feature: K): SplitNode[K, V, T, Unit] =
+    SplitNode(feature, predicate, LeafNode(0, leftDistribution), LeafNode(1, rightDistribution))
 }
 
+
 /** Evaluates the goodness of a candidate split */
 trait Evaluator[V, T] {
   /** returns a (possibly transformed) version of the input split, and a numeric goodness score */
-  def evaluate(split: Split[V, T]): (Split[V, T], Double)
+  def evaluate(split: Split[V, T]): Option[(Split[V, T], Double)]
```

Comment:
May be a question for @avibryant too, but what does `None` mean and could we put that in the docs?